features/assets.feature

```diff
@@ -15,18 +29,38 @@ Feature: Assets
     When I request "/assets/513a0efbed915d425e000002"
     And I should see "120613_Albania_Travel_Advice_WEB_Ed2_jpeg.jpg"
```

Comment:
Looks like a lot of these scenarios could do with syntax being tidied up - the assertions should being with "then", possibly followed by "and"s.

Reply:
Good spot. I've found lots of these little issues as I've been going through the tests. They're out-of-scope for this PR, but I'm in the area in one of the follow-up PRs I've got planned, then I'll look at this.